hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestHarness.java

```diff
@@ -225,6 +229,22 @@ protected void initMetaClient(HoodieTableType tableType) throws IOException {
     metaClient = HoodieTestUtils.init(hadoopConf, basePath, tableType);
   }
 
+  protected Properties getPropertiesForKeyGen() {
```

Comment:
HoodieTestDataGenerator has these fields in the commonly used schema and hence hardcoded it here, so that all tests can call into this to generate props.